I"|<p>PG::ObjectInUse: ERROR: データベース"database_name"は他のユーザからアクセスされています</p>
:ET